udata_gouvfr/migrations/2017-08-04-fix-wrong-spatial-coverages.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * Fix wrong spatial coverage
+ *
+ * As a side-effect from the early udata spatial completion
+ * some dataset are using Canton/Arrondissement/Iris
+ * as spatial coverage instead of cities.
+ *
+ * This migration is built from a production extract and reaffect
+ * dataset to their town spatial coverage.
+ *
+ * Some of them are not relevant and are just pulled from the spatial coverage
+ * (these are unpublished datasets and datasets with multiple level declarations)
+ *
+ * Some datasets are handled cas by case (some dataset have)
+ *
+ * NOTE: this migration only works before the new geozones identifier are used
```

Comment:
The other one is the `udata territories migrate_zone_ids` command and so it will run after and can also be run multiple times :+1: